skin.json

```diff
@@ -123,8 +123,14 @@
   },
   "controlBar": {
     "iconStyle": {
-      "color": "#8E8E8E",
-      "opacity": 1
+      "active": {
+        "color": "#FFFFFF",
+        "opacity": 1
+      },
+      "inactive": {
+        "color": "#8E8E8E",
+        "opacity": 0.6
+      }
```

Comment:
@mistermendez, it looks like we forgot about moreOptions iconStyle. We should make the same change there as well.